### DISCUSSION TITLE
[fastapi docs] [Pydantic BaseModel] request example missing

### DISCUSSION THREAD

**Discussion by Yukun-Cui** [fastapi/fastapi#11370] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from pydantic.v1 import *

self._router.add_api_route(
    "/v1/embeddings",
    self.create_embedding,
    methods=["POST"],
    dependencies=(
        [Security(self._auth_service, scopes=["models:read"])]
        if self.is_authenticated()
        else None
    ),
====================================================================================
async def create_embedding(self, request: Request) -> Response:
    body = CreateEmbeddingRequest.parse_obj(await request.json())
    model_uid = body.model

    try:
        model = await (await self._get_supervisor_ref()).get_model(model_uid)
    except ValueError as ve:
        logger.error(str(ve), exc_info=True)
        await self._report_error_event(model_uid, str(ve))
        raise HTTPException(status_code=400, detail=str(ve))
    except Exception as e:
        logger.error(e, exc_info=True)
        await self._report_error_event(model_uid, str(e))
        raise HTTPException(status_code=500, detail=str(e))

    try:
        embedding = await model.create_embedding(body.input)
        return Response(embedding, media_type="application/json")
    except RuntimeError as re:
        logger.error(re, exc_info=True)
        await self._report_error_event(model_uid, str(re))
        self.handle_request_limit_error(re)
        raise HTTPException(status_code=400, detail=str(re))
    except Exception as e:
        logger.error(e, exc_info=True)
        await self._report_error_event(model_uid, str(e))
        raise HTTPException(status_code=500, detail=str(e))
====================================================================================
class CreateEmbeddingRequest(BaseModel):
    model: str
    input: Union[str, List[str], List[int], List[List[int]]] = Field(
        description="The input to embed."
    )
    user: Optional[str] = None

    class Config:
        schema_extra = {
            "example": {
                "input": "The food was delicious and the waiter...",
            }
        }
```

### Description

I want to declare the request example in request body like that
<img width="1419" alt="317680907-449a0b74-2535-49f5-a91e-a0e63490ab50" src="https://github.com/tiangolo/fastapi/assets/102746679/59fd7223-7ccf-4325-aebe-2a3eef3ecb80">
but the result of my code is like that
<img width="1393" alt="317680389-5c43cc57-feb8-493d-9848-232861a23db4" src="https://github.com/tiangolo/fastapi/assets/102746679/e140975b-3d0e-4bcd-a474-5337010c9ccb">
Who can help me, thanks in advance.

### Operating System

Linux

### Operating System Details

CentOS 7

### FastAPI Version

0.109.2

### Pydantic Version

2.6.4

### Python Version

3.10.13

### Additional Context

my Pydantic Version is V2，but my code import ``pydantic.v1``

**Comment by YuriiMotov** [comment]
Sorry, but your code looks like you are using not FastAPI, but something else..
Have you read FastAPI documentation? FastAPI has the best documentation ever. Just find time to read it!

**Comment by jgould22** [comment, marked as answer]
FastAPI uses the parameters of the controller function to generate these examples.

You are using the Request object as a parameter and thus FastAPI has no way to know what is in it.

Here are some ways you can get the examples with fastapi 

https://fastapi.tiangolo.com/tutorial/schema-extra-example/

**Reply by Yukun-Cui** [reply]
```
    async def create_embedding(self, body: CreateEmbeddingRequest) -> Response:
        # body = CreateEmbeddingRequest.parse_obj(await request.json())
        model_uid = body.model
```

Thank you for help me, I changed my code as you say, but there is something error:

![1](https://github.com/tiangolo/fastapi/assets/102746679/f5a811ff-8643-4b5d-8b2e-235cbf8c94d9)
